babybear/src/main/java/frc/robot/subsystems/WristSubsystem.java

```diff
@@ -0,0 +1,172 @@
+    /**********************************************************************
+    CONSTRUCTORS
+    ************************************************************************/
+
+    /**
+    * constructor for WristSubsystem subsystem
+    */
+    public WristSubsystem() {
+      CommandScheduler.getInstance().registerSubsystem(this);
+    }
+
+
+
+    /**********************************************************************
+    PUBLIC METHODS
+    ************************************************************************/
+    /**
+     * A method to set the wrist motor to a certain RPM based on a relative speed input
+     * @param everyBotPickerSpeed the relative speed -1.0 to 1.0 to run the everyBot arm motor at
+     */
+
+    public void setPickerRelativeSpeed(double everyBotPickerSpeed) {
+      this.requestedWristMotorSpeed = MotorUtils.truncateValue(everyBotPickerSpeed, -1.0, 1.0);
+      wristMotor.set(this.requestedWristMotorSpeed);
+    }
+
+    public void setPickerAngle(double wristAngle) { 
+        double targetPositionTicks = wristAngle * TICKS_PER_DEGREE;
+        double curPosition = wristEncoder.getPosition();
+        
+        if (Math.abs(curPosition - targetPositionTicks) > TOLERANCE) {
+            // If the current position is outside the tolerance range, use the PID controller to move the wrist to the target position
+            wristPidController.setReference(targetPositionTicks, ControlType.kPosition);
+        } else {
+            // If within tolerance, stop the wrist motor
+            wristMotor.set(0);
+        }
+    }
+
+    
+    
+    @Override
+    public void periodic() {
+      // confirm that the smart motion is setup - no-op after it is setup first time
+      this.initializeMotorsSmartMotion();
+      this.refreshWristPosition();
+      //wristMotor.set(this.requestedWristMotorSpeed * this.wristMotorSpeedReductionFactor);
+
+    }
+
+    @Override
+    public void simulationPeriodic() {
+
+    }
+
+    /* *********************************************************************
+    PRIVATE METHODS
+    ************************************************************************/
+    /**
+     * A function intended to be called from periodic to update encoder value of the motor.
+     */
+    private void refreshWristPosition() {
+      SmartDashboard.putNumber("EveryArmMotorSpeedRpm", this.wristEncoder.getVelocity());
+      SmartDashboard.putNumber("Wrist Angle", this.wristEncoder.getPosition());
+    }
+
+    // a method devoted to establishing proper startup of the jaws motors
+    // this method sets all of the key settings that will help in motion magic
+    private void initializeMotorsSmartMotion() {
+      if(this.motorInitializedForSmartMotion == false) { 
+        // PID coefficients
+        kP = 2e-4; 
+        kI = 0;
+        kD = 0;
+        kIz = 0; 
+        kFF = 0.00001; 
+        kMaxOutput = 1; 
+        kMinOutput = -1;
+        maxRPM = Constants.neoFiveFiveZeroMaximumRevolutionsPerMinute;
```

Comment:
ok ... this is for a 550 neo ... not the larger neo that will be used for the wrist ... at least I'm under the impression that the large neo will be used for the wrist ... you tell me

Reply:
you want ... neoMaximumRevolutionsPerMinute if we are using the larger (more powerful neos)